lib/numorstring/port.go

```diff
@@ -50,13 +51,16 @@ func PortFromString(s string) (Port, error) {
 
 	parts := strings.Split(s, ":")
 	if len(parts) != 2 {
-		return Port{}, errors.New("invalid port format")
+		msg := fmt.Sprintf("invalid port format (%s)", s)
+		return Port{}, errors.New(msg)
```

Comment:
it can - i think having it on a separate line is easier to read tho.

I'll change it if its bugging you or if it is against golang suggested practices.  Let me know.